### PR TITLE
replace ext_pylintrc with latest pylintrc in azure-cli-extensions repo

### LINKED ISSUE #305: Latest release (v0.1.33) breaks pylint in custom extension repos (not azure-cli-extensions)
#301 broke `azdev style [ext]` in extensions not hosted in azure-cli-extensions because it falls back to the [ext_pylintrc](https://github.com/Azure/azure-cli-dev-tools/blob/dev/azdev/config/ext_pylintrc) file.

This file **does not** ignore the `generated`, `vendored_sdks`, `privates` folders, thus causing the `azdev style` command to run pylint on these folders with no way to override/ignore failures.

I'm using [python autorest](https://github.com/Azure/autorest.python) (v5.8.0 and several others) to generate an SDK in the `vendored_sdk` directory, and currently the generated code doesn't pass the pylint rules.

The only workaround I've found is to manually modify the ext_pylintrc file in the .azdev directory.

cc @jiasli @kairu-ms

### PR DESCRIPTION
**Pull request by colbylwilliams**
fixes #305 - Latest release (v0.1.33) breaks pylint in custom extension repos (not azure-cli-extensions)

---

This PR replaces the "fallback" ext_pylintrc with latest pylintrc in azure-cli-extensions repo

---

#301 broke `azdev style [ext]` in extensions not hosted in azure-cli-extensions because it falls back to the [ext_pylintrc](https://github.com/Azure/azure-cli-dev-tools/blob/dev/azdev/config/ext_pylintrc) file.

This file **does not** ignore the `generated`, `vendored_sdks`, `privates` folders, thus causing the `azdev style` command to run pylint on these folders with no way to override/ignore failures.

I'm using [python autorest](https://github.com/Azure/autorest.python) (v5.8.0 and several others) to generate an SDK in the `vendored_sdk` directory, and currently the generated code doesn't pass the pylint rules.

The only workaround I've found is to manually modify the ext_pylintrc file in the .azdev directory.

cc @jiasli @kairu-ms

### CODE REVIEW

**Review by kairu-ms — APPROVED** [review]
LGTM

**Comment by colbylwilliams** [comment]
@jiasli @kairu-ms do you have a estimate for when this change can be merged and released?  We had to disable linting in our CI, and would like to re-enable once this is released.  Thanks

File: azdev/config/ext_pylintrc [hunk 1]
```diff
@@ -1,17 +1,14 @@
 [MASTER]
+ignore=tests,generated,vendored_sdks,privates
 ignore-patterns=test_*
```
Comment (jiasli) [inline azdev/config/ext_pylintrc]: I think we can remove this line as it is simply wrong (https://github.com/Azure/azure-cli/pull/18003#discussion_r628096102).

```suggestion
```

**Review by jiasli — APPROVED** [review]

**Comment by jiasli** [comment]
Site note: this file should be in sync with https://github.com/Azure/azure-cli-extensions/blob/bad8f3bc7b216a252331c313880c5e85bf0963c0/pylintrc

**Comment by colbylwilliams** [comment]
@jiasli @kairu-ms can one of you please merge?  I don't have write access to this repo.  Also what is your release schedule so I know when to expect this change to be included in a release?

**Comment by colbylwilliams** [comment]
@jiasli @kairu-ms please merge.

**Pull request merged**